charts/metrics-server/templates/clusterrole.yaml

```diff
@@ -17,6 +17,7 @@ rules:
     resources:
       - pods
       - nodes
+      - nodes/stats
```

Comment:
Hi, @nlamirault, Why do we need to add this permission?
For metrics-server 0.6.x, we access the nodes/metrics endpoint